### DISCUSSION TITLE
Ocra Profiles

### DISCUSSION THREAD

**Discussion by crazyslicster** [OpenNeptune3D/OpenNept4une#144] [General]
Does anyone know if there is active work on more OpenNept4une Orca Slicer profiles?

Seems that the new version of Orca slicer has the Neptune 4 profiles so wondering if those should still working with the OpenNept4une firmware.  I assume since the hardware remains the same, they should work?

Thanks

**Comment by halfmanbear** [comment]
The start gcode is unique in our profiles although these can be copied to any profile you currently have

**Reply by crazyslicster** [reply]
Since they have all the different preset sizes, maybe this would be good to copy to the github repos with the copied gcode?  I've never really contributed to github but would be happy to copy them over.
